core/spec/models/spree/order/payment_spec.rb

```diff
@@ -161,6 +161,22 @@ module Spree
         order.payment_total = 10.20
         expect(order.outstanding_balance).to be_within(0.001).of(-2.00)
       end
+      it 'should incorporate refund reimbursements' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.